src/test/java/org/broadinstitute/consent/http/service/DatasetServiceTest.java

```diff
@@ -62,7 +66,7 @@ public void setUp() {
     }
 
     private void initService() {
-        datasetService = new DatasetService(consentDAO, datasetDAO, userRoleDAO, useRestrictionConverter);
+        datasetService = new DatasetService(consentDAO, datasetDAO, userRoleDAO, dataSetAuditDAO, datasetAssociationDAO, useRestrictionConverter);
     }
 
     @Test
```

Comment:
I'd love to see new tests for all of the methods moved to the service class. We need to incrementally boost our test coverage especially across the service classes.